client/template/search.js

```diff
@@ -39,6 +39,7 @@ Template.ChatpalSearch.onRendered(function() {
 				console.log('ping failed');
 				this.enabled.set(false);
 			} else {
+				this.enabled.set(true);
```

Comment:
Once a result could be retrieved, this reactive var needs to be reset in order to make the results being displayed